### PR TITLE
Opt-in Logcat Bundle based on the client functionality

### PR DESCRIPTION
**Pull request by jmatsu**
Logcat Bundle produces insufficient Logcat on unsupported client

### CODE REVIEW

File: sdk/src/main/java/com/deploygate/sdk/Compatibility.java [hunk 1]
```diff
@@ -5,14 +5,26 @@ final class Compatibility {
      * All values must be *inclusive*. {@link Integer#MAX_VALUE} means the version is not fixed yet.
      */
     static final class ClientVersion {
-        /**
-         * older clients crash due to ClassNotFound if SDK sends custom exceptions.
-         */
+        static final int SUPPORT_UPDATE_MESSAGE_OF_BUILD = 39;
         static final int SUPPORT_SERIALIZED_EXCEPTION = 42;
+        static final int SUPPORT_LOGCAT_BUNDLE = Integer.MAX_VALUE;
```
Comment (tnj) [inline sdk/src/main/java/com/deploygate/sdk/Compatibility.java:10]: Will be determined later, right?

**Review by tnj — APPROVED** [review]
LGTM!

File: sdk/src/main/java/com/deploygate/sdk/Compatibility.java (hunk 1 above)
Comment (jmatsu) [reply sdk/src/main/java/com/deploygate/sdk/Compatibility.java:10]: Yes~

**Comment by jmatsu** [comment]
Thank you for your quick review!

**Pull request merged**